.github/workflows/submodule_update.yml

```diff
@@ -0,0 +1,38 @@
+name: submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+
+jobs:
+  tarantool-ee:
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: 'tarantool/tarantool-ee'
+      FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
+      CHECKOUT_BRANCH: ${{ github.ref_name }}
+      PR_AGAINST_BRANCH: ${{ github.ref_name }}
+      PR_TITLE_PREFIX: ${{ github.ref_name == 'master' && '' ||
+        format('[{0}] ', github.ref_name) }}
+      PR_TITLE: 'tarantool: bump to new version'
+      COMMIT_MESSAGE: |
+        tarantool: bump to new version
+      
+        NO_DOC=submodule update
+        NO_TEST=submodule update
+        NO_CHANGELOG=submodule update
+
+    steps:
+      - name: Create PR with submodule update
+        uses: tarantool/actions/update-submodule@master
+        with:
+          github_token: ${{ secrets.EE_UPDATE_SUBMODULE_TOKEN }}
```

Comment:
Looks like you forgot `submodule: ${{ env.SUBMODULE }}`.

Reply:
done